package.json

```diff
@@ -706,7 +706,9 @@
       "tsx>esbuild": false,
       "@metamask/eth-trezor-keyring>@trezor/connect-web>@trezor/connect>@trezor/protobuf>protobufjs": false,
       "firebase>@firebase/firestore>@grpc/proto-loader>protobufjs": false,
-      "@lavamoat/lavadome-core>@lavamoat/preinstall-always-fail": false
+      "@lavamoat/lavadome-core>@lavamoat/preinstall-always-fail": false,
+      "@metamask/desktop>ws>bufferutil": false,
```

Comment:
Were these changes included by mistake?

Reply:
@Gudahtt no the pipeline fails if they are not included

Reply:
But we aren't using the `@metamask/desktop` package anywhere

Reply:
@Gudahtt right make sense I just removed them